<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowAdmin</class>
 <widget class="QMainWindow" name="WindowAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WindowAdmin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutMenu" stretch="0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutButtons">
        <item>
         <widget class="QLabel" name="LogoRD">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>:/images/images/RD_logo-e1392478122173.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonNewRace">
          <property name="text">
           <string>New Race</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonAddEditTeam">
          <property name="text">
           <string>Add/Edit Team</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonStats">
          <property name="text">
           <string>Stats</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="CurrentWindow"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAuthors"/>
    <addaction name="actionQT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Authors</string>
   </property>
  </action>
  <action name="actionQT">
   <property name="text">
    <string>QT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
